README.md

```diff
@@ -1,2 +1,24 @@
-# hello-microverse1
-This repo was made for the setting up of linters.
+# Hello Microverse
+This project was actualised for the setup of my linters.
+
+# Additional description about the project and its features.
+
+# Built With
+1. HTML
+2. CSS
+3. Linters (Webhint, Stylelint, Lighthouse)
+
+
+GitHub: @direakanbi
+Twitter: @direakanbi
+LinkedIn: Akanbi Dire.
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] In the readme file, add the necessary steps to follow to get started locally. How will someone without computer and coding proficiency be able to use install and use your app/repo